Financial accounting -> transactions, update to Primeagen talk (#1759)
tigerbeetle/tigerbeetle: README.md

### Using the CLI Client

Now that you've got a cluster running, let's connect to it and do some

> transacting!

Reviewer: ```suggestion
accounting!
```

## Next Steps

> Watch an introduction to TigerBeetle on [ThePrimeagen](https://www.youtube.com/watch?v=sC1B3d9C_sI) for our design

Reviewer: ```suggestion
Watch an introduction to TigerBeetle on [The Primeagen](https://www.youtube.com/watch?v=sC1B3d9C_sI) for our design
```

> decisions regarding performance, safety, and financial transactions

Reviewer: ```suggestion
decisions regarding performance, safety, and financial accounting debit/credit
```

I think I went too far... ;)
